<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dashboard - Aplikasi Jasa Makeup</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f5f5f5;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frameSidebar">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frameSidebar {
    background-color: #E91E63;
}

QPushButton {
    background-color: transparent;
    color: white;
    text-align: left;
    padding-left: 20px;
    border: none;
    border-radius: 0px;
    font-size: 11pt;
}

QPushButton:hover {
    background-color: #C2185B;
}

QPushButton:pressed {
    background-color: #AD1457;
}

QLabel {
    color: white;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lblLogo">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>üíÑ MAKEUP APP</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDashboard">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üè† Dashboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPelanggan">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üë• Data Pelanggan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLayanan">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>‚ú® Layanan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnJadwal">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üìÖ Jadwal Booking</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTransaksi">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üõí Transaksi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPembayaran">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üí≥ Pembayaran</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetContent" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frameHeader">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame#frameHeader {
    background-color: white;
    border-bottom: 1px solid #ddd;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lblPageTitle">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Dashboard</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lblUsername">
            <property name="text">
             <string>üë§ Admin User</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnLogout">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #f44336;
    color: white;
    border: none;
    border-radius: 5px;
    padding: 8px 15px;
}

QPushButton:hover {
    background-color: #da190b;
}</string>
            </property>
            <property name="text">
             <string>Logout</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="pageContent" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget#pageContent {
    background-color: #f5f5f5;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="widgetStats" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>120</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>120</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QFrame" name="frameStat1">
               <property name="styleSheet">
                <string notr="true">QFrame {
    background-color: white;
    border-radius: 8px;
    border: 1px solid #ddd;
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="lblStatTitle1">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: #666;</string>
                  </property>
                  <property name="text">
                   <string>Total Pelanggan</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lblStatValue1">
                  <property name="font">
                   <font>
                    <pointsize>24</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: #333;</string>
                  </property>
                  <property name="text">
                   <string>0</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frameStat2">
               <property name="styleSheet">
                <string notr="true">QFrame {
    background-color: white;
    border-radius: 8px;
    border: 1px solid #ddd;
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QLabel" name="lblStatTitle2">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: #666;</string>
                  </property>
                  <property name="text">
                   <string>Transaksi Bulan Ini</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lblStatValue2">
                  <property name="font">
                   <font>
                    <pointsize>24</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: #333;</string>
                  </property>
                  <property name="text">
                   <string>0</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frameStat3">
               <property name="styleSheet">
                <string notr="true">QFrame {
    background-color: white;
    border-radius: 8px;
    border: 1px solid #ddd;
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QLabel" name="lblStatTitle3">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: #666;</string>
                  </property>
                  <property name="text">
                   <string>Pendapatan Hari Ini</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lblStatValue3">
                  <property name="font">
                   <font>
                    <pointsize>20</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: #333;</string>
                  </property>
                  <property name="text">
                   <string>Rp 0 Jt</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frameStat4">
               <property name="styleSheet">
                <string notr="true">QFrame {
    background-color: white;
    border-radius: 8px;
    border: 1px solid #ddd;
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QLabel" name="lblStatTitle4">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: #666;</string>
                  </property>
                  <property name="text">
                   <string>Jadwal Hari Ini</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lblStatValue4">
                  <property name="font">
                   <font>
                    <pointsize>24</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: #333;</string>
                  </property>
                  <property name="text">
                   <string>0</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupJadwal">
            <property name="styleSheet">
             <string notr="true">QGroupBox {
    border: 2px solid #E91E63;
    border-radius: 8px;
    margin-top: 10px;
    padding: 15px;
    background-color: white;
    font-weight: bold;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 10px;
    background-color: white;
    color: #E91E63;
}</string>
            </property>
            <property name="title">
             <string>üìÖ Jadwal Hari Ini</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QTableWidget" name="tableJadwalDashboard">
               <property name="styleSheet">
                <string notr="true">QTableWidget {
    background-color: white;
    border: 1px solid #ddd;
    gridline-color: #f0f0f0;
}

QTableWidget::item:selected {
    background-color: #FCE4EC;
    color: #333;
}

QHeaderView::section {
    background-color: #E91E63;
    color: white;
    padding: 8px;
    border: none;
    font-weight: bold;
}</string>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <column>
                <property name="text">
                 <string>Waktu</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Pelanggan</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Layanan</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Status</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
